discovery-provider/Dockerfile

```diff
@@ -79,6 +79,9 @@ RUN python3 -m pip install --upgrade pip && \
 COPY requirements.txt requirements.txt
 RUN python3 -m pip install -r requirements.txt --no-cache-dir
 
+COPY es-indexer es-indexer
```

Comment:
I think we should move this to line 76, but I assume you'll make more changes over the next few weeks than the rest of the discovery team changing [requirements.txt](https://github.com/AudiusProject/audius-protocol/commits/master/discovery-provider/requirements.txt).

Once you're done with the bulk of your work on es-indexer, we should move this to line 76, but for now let's give you faster builds. :)